extensions/amp-story/1.0/test/test-amp-story.js

```diff
@@ -1709,6 +1709,77 @@ describes.realWin(
           'Swipe up'
         );
       });
+
+      describe('remote locales', () => {
+        beforeEach(() => {
```

Comment:
`before` should be enough
Also, plz toggle the experiment back to `false` after running these tests